config/jobs/periodic/docker-in-docker/periodic-all-in-one.yaml

```diff
@@ -0,0 +1,29 @@
+periodics:
+  - name: docker-all-in-one-test
+    labels:
+      preset-secrets: "true"
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    interval: 2h
+    spec:
+      containers:
+      - image: quay.io/alunsin/all_in_one_dind
```

Comment:
remind me again why we're not using https://hub.docker.com/_/docker? did it not exist when we started this b/c we didn't have docker official support for power at the time?